homeassistant/components/samsungtv/entity.py

```diff
@@ -89,10 +89,21 @@ async def _async_turn_off(self) -> None:
     async def _async_turn_on(self) -> None:
         """Turn the remote on."""
         if self._turn_on_action:
+            LOGGER.info("Attempting to turn on %s via automation", self.entity_id)
```

Comment:
```suggestion
            LOGGER.debug("Attempting to turn on %s via automation", self.entity_id)
```

If they setup an automation we probably don't need to log at info as its a conscious choice